dev/grafana/provisioning/dashboards/filter-processing-dashboard.json

```diff
@@ -0,0 +1,383 @@
+{
```

Comment:
What about aggregate the dashboards to map each components:
- fleet manager dashboard
- fleet shard dashboard (let's focus on single instance only because I expect the target env will have some mechanism to aggregate metrics)
- executor dashboard

Wdyt?